app/static/tests/unit/components/options/parameterSetView.test.ts

```diff
@@ -286,8 +286,7 @@ describe("ParameterSetView", () => {
         expect(displayNameInput.element.value).toBe("Set 1");
         expect(displayNameText.isVisible()).toBe(false);
 
-        // 4 on mount + 1 input on mount + 4 after update
-        expect(mockTooltipDirective).toHaveBeenCalledTimes(9);
+        // 4 on mount + 1 input on mount + 2nd one is saveIcon
```

Comment:
Can remove this comment if you don't want to do a check on this total number of calls any more.

Reply:
oh i updated that comment as a justification for why we are looking at `mock.calls[6]` cos its the 7th one